index.html

```diff
@@ -9,24 +9,54 @@
   </head>
   <body>
     <header>
-      <nav class="nav-mobile nav-bar">
-        <h2>John Omachi</h2>
-        <img src="./images/hamburger.png" alt="hamburger icon" />
+      <nav class="mobile-view">
+        <div class="nav-mobile nav-bar">
+          <h2 id="logoMobile">John Omachi</h2>
+          <div>
+            <img
+              src="./images/hamburger.png"
+              id="hamburger"
+              alt="hamburger icon"
+            />
+            <img
+              src="./images/close.png"
+              class="hidden close"
+              id="closeX"
+              alt=""
+            />
+          </div>
+        </div>
+        <div id="menu" class="hidden menuList">
+          <ul class="menuListUl">
+            <li id="menu-link1" class="menuLinks">
+              <a href="#works">Portfolio</a>
+              <img src="./images/next.png" alt="next icon" />
+            </li>
+            <li id="menu-link2" class="menuLinks">
+              <a href="#about">About</a>
+              <img src="./images/next.png" alt="next icon" />
+            </li>
+            <li id="menu-link3" class="menuLinks">
+              <a href="#contact">Contact</a>
```

Comment:
- [ ] kindly make your menu links to take you to their respective different page sections. This will ease navigation. Nice work 😄